src/PipelineBuilderInterface.php

```diff
@@ -1,23 +1,17 @@
 <?php
+declare(strict_types=1);
```

Comment:
If I remember correctly interfaces do not need the `declare` declaration since no code is executed in these files anyway

Reply:
That *may* be the case, but I think it is better to have it everywhere for consistency.

Reply:
No value is lost or added with the removal or inclusion of this line of code 🤷‍♂️ it's all good for me